pkg/reconciler/shared/tektonconfig/upgrade/pre_upgrade.go

```diff
@@ -103,3 +103,33 @@ func removeDeprecatedAddonParams(ctx context.Context, logger *zap.SugaredLogger,
 	_, err = operatorClient.OperatorV1alpha1().TektonConfigs().Update(ctx, tcCR, metav1.UpdateOptions{})
 	return err
 }
+
+// previous version of the Tekton Operator does not install the TektonResult via
```

Comment:
```suggestion
// previous version of the Tekton Operator does not install the TektonResult via TektonConfig
```

It would be good if you cover the unit test

but yeah you do it in a followup PR